src/dms/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -103,12 +103,19 @@ private async Task<List<IAPIResponse>> ProcessDataTable(string entityType, DataT
 
             foreach (var descriptor in dataTable.ExtractDescriptors())
             {
-                _apiResponses.Add(
-                    await _playwrightContext.ApiRequestContext?.PostAsync(
-                        $"{baseUrl}/{descriptor["descriptorName"]}",
-                        new() { DataObject = descriptor }
-                    )!
-                );
+                var response = await _playwrightContext.ApiRequestContext?.PostAsync(
+                    $"{baseUrl}/{descriptor["descriptorName"]}",
+                    new() { DataObject = descriptor }
+                )!;
+                _apiResponses.Add(response);
+
+                response
+                    .Status.Should()
+                    .BeOneOf(
+                        OkCreated,
+                        $"POST request for {entityType} descriptor {descriptor["descriptorName"]} failed:\n{response.TextAsync().Result}"
+                    );
```

Comment:
This code runs in for `Given` statements. Previously we were ignoring errors - now an error will cause a failure early on, which is easier to diagnose.

Reply:
Excellent